src/Place/Pages/PPA001Places/Places.tsx

```diff
@@ -0,0 +1,42 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useContext, useEffect } from 'react';
+import { ActivityIndicator } from 'react-native-paper';
+
+import { PPA001PlacesController } from './PlacesController';
+
+import { BottomTabParamList } from '@/App';
+import { PCO001SearchPlace } from '@/Place/Components/PCO001SearchPlace/SearchPlace';
+import { PCT011MPlacesList } from '@/Place/Models/PDB01MPlace/Contexts/PCT011MPlacesList';
+import { PMC01101GoogleMapPlacesList } from '@/Place/Models/PDB01MPlace/Contexts/PCT011MPlacesList/ModelComponents/PMC01101GoogleMapPlacesList/GoogleMapPlacesList';
+import { PMC01102PlacesList } from '@/Place/Models/PDB01MPlace/Contexts/PCT011MPlacesList/ModelComponents/PMC01102PlacesList/PlacesList';
+
+export const PPA001Places = ({ navigation, route }: NativeStackScreenProps<BottomTabParamList, 'PPA001Places'>) => {
+	const { onAutoCompleteClicked, onPlaceSelected } = PPA001PlacesController();
+	const { setAddress, isFirstLoading } = useContext(PCT011MPlacesList);
+	const { country, administrativeAreaLevel1, administrativeAreaLevel2, locality } = route.params;
+
+	useEffect(() => {
+		if (country === '') {
+			// TODO: https://github.com/Ayato-kosaka/spelieve/issues/305 現在地からGepoint取得 
+			setAddress({
+				country: '日本',
+				administrativeAreaLevel1: '神奈川県',
+				administrativeAreaLevel2: '',
+				locality: '横浜市',
+			});
+		} else {
+			setAddress({ country, administrativeAreaLevel1, administrativeAreaLevel2, locality });
+		}
```

Comment:
country === '' のときは何も取れない動きにしたいから↓みたいになると思う！
```suggestion
		if (country) {
			setAddress({ country, administrativeAreaLevel1, administrativeAreaLevel2, locality });
		} else {
			// TODO: https://github.com/Ayato-kosaka/spelieve/issues/305 現在地からGepoint取得 
			setAddress({
				country: '日本',
				administrativeAreaLevel1: '神奈川県',
				administrativeAreaLevel2: '',
				locality: '横浜市',
			});
		}
```

Reply:
[5a84647](https://github.com/Ayato-kosaka/spelieve/pull/173/commits/5a84647d37c8cf5ba16f2b54f86452b936d6d9f8)で対応